package.go

```diff
@@ -21,4 +21,6 @@ type Package struct {
 	// correctly ordered when compared with other representations from the same
 	// producer.
 	NormalizedVersion Version `json:"normalized_version,omitempty"`
+	// Module and stream which this package is part of
+	Module string `json:"module,omitempty"`
```

Comment:
is it okay to couple module and stream into a single string? Or should they be two separate fields?

Reply:
I was thinking about it, but I didn't find a use-case of having it in separate fields. Basically it is very similar to a version property (it also contains epoch, version and release, but it is represented as a simple string)